modules/gapi/include/opencv2/gapi/infer/onnx.hpp

```diff
@@ -220,11 +220,29 @@ struct GAPI_EXPORTS_W_SIMPLE OpenVINO {
         return *this;
     }
 
+    /** @brief Configures OpenVINO Execution Provider parameters using a map of options.
+     *
+     * This function allows setting multiple configuration options for the OpenVINO Execution Provider
+     * by passing a map of key-value pairs. It updates the ParamsMap with the new parameters.
```

Comment:
User should not know about `ParamsMap` as it's our implementation detail

Reply:
Yes removed.